db/products.js

```diff
@@ -0,0 +1,82 @@
+const productCollection = {
+  products: [],
+};
+let productId = 1;
+
+function create(obj) {
+  const hasKeys = obj.name && obj.price && obj.inventory;
+
+  if (hasKeys) {
+    obj.id = productId;
+    productCollection.products.push(obj);
+    productId++;
+    return `{ "success": true}`;
+  } else if (!hasKeys) {
```

Comment:
This looks like it should be an `else` rather than an `else if` since it's a binary condition.